### PR TITLE
refs #9403 - get rhsm certificate from rhsm configration

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: src/katello/agent/katelloplugin.py [hunk 1]
```diff
@@ -117,10 +117,11 @@ def setup_plugin():
     cfg = plugin.cfg()
     rhsm_conf = Config(RHSM_CONFIG_PATH)
     certificate = ConsumerIdentity.read()
+    cfg.messaging.cacert = rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']
```
Comment (jlsherrill) [inline src/katello/agent/katelloplugin.py]: Does this handle the string substitution from rhsm.conf?: 
repo_ca_cert = %(ca_cert_dir)skatello-server-ca.pem

Comment (jlsherrill) [inline src/katello/agent/katelloplugin.py]: Does the Config() object handle that for you?

Comment (stbenjam) [inline src/katello/agent/katelloplugin.py]: Unfortunately `Config` doesn't do it, which is why I'm doing the substitution with `rhsm_conf['rhsm']`

I wonder if we can't get this hooked up to Jenkins? Not urgent, I ran the tests locally and they pass.  I need to get a jenkins account one of these days...

Comment (jlsherrill) [inline src/katello/agent/katelloplugin.py]: ohhhh i see, i was a tad confused there.

**Comment by jlsherrill** [comment]
ACK from me

**Comment by jlsherrill** [comment]
and +1 to getting jenkins test runs.  I'll see about setting up a job and we can point the PR processor at it i think

**Comment by stbenjam** [comment]
@jlsherrill It needs a bunch of dependencies, it took a bit to figure out, but it was mostly python-gofer and pulp stuff.  

If those are installable with pip on Ubuntu, we could use Travis.

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**